environments/backfillEvents.toml

```diff
@@ -63,7 +63,7 @@
         type = "eth_event"
         repository = "github.com/makerdao/vdb-mcd-transformers"
         migrations = "db/migrations"
-        contracts = ["MCD_CAT", "MCD_FLAP_1.0.0", "MCD_FLAP_1.0.9", "MCD_FLIP_BAT_A_1.0.0", "MCD_FLIP_BAT_A_1.0.9",
+        contracts = ["MCD_CAT_1.0.0", "MCD_FLAP_1.0.0", "MCD_FLAP_1.0.9", "MCD_FLIP_BAT_A_1.0.0", "MCD_FLIP_BAT_A_1.0.9",
```

Comment:
should add new cat address to all event transformers where the old one is relevant